src/mcts/stoppers/alphazero.cc

```diff
@@ -73,8 +60,7 @@ std::unique_ptr<SearchStopper> AlphazeroTimeManager::GetStopper(
 
   auto total_moves_time = *time + increment - move_overhead_;
```

Comment:
You shouldn't be adding the increment here either.

Reply:
Ehm. Can you explain where the increment would be used if it would also be removed here?

Reply:
Nowhere really, since we don't try to predict number of remaining moves to estimate total game duration. Note that uci time control is not like in fide rules, so the increment is not added to the current move available time.

Reply:
ah, that clarifies